src/editor/contentState/selectionCtrl.js

```diff
@@ -0,0 +1,28 @@
+import selection from '../selection'
+
+const selectionCtrl = ContentState => {
+  ContentState.prototype.selectionHandler = function (event) {
+    const { start } = this.cursor
+    const block = this.getBlock(start.key)
+    if (/th|td/.test(block.type)) {
+      event.preventDefault()
+      console.log('table detected!') // TODO: #DEBUG
+
+      // @Jocs Selects nothing...
+      selection.setCursorRange({
```

Comment:
We should not use `setCursorRange` directly, `render` or `partialRender` will help us to render the cursor. all we need to do is set the cursor by `this.cursor = newCursor`.